scripts/configs/config_birdCOM.yaml

```diff
@@ -0,0 +1,213 @@
+data:
+  # resize dimensions to streamline model creation
+  image_resize_dims:
+    height: 256
+    width: 512
+  # ABSOLUTE path to data directory
+  data_dir: "/home/selmaan/Documents/GitHub/lightning-pose/data/Chickadee"
+  # ABSOLUTE path to unlabeled videos' directory
+  video_dir: "/home/selmaan/Documents/GitHub/lightning-pose/data/Chickadee/videos"
+  # location of labels; this should be relative to `data_dir`
+  csv_file: COM_Labels.csv
+  # downsample heatmaps - 2 | 3
+  downsample_factor: 2
+  # total number of keypoints
+  num_keypoints: 3
+  # keypoint names
+  keypoint_names:
+    - head
+    - body
+    - tail
+  # indicator to use 2-stage dynamic crop algorithm, False/True
+  dynamic_crop: False
+  # max allowed number of instances, i.e. 1 for single-animal, >1 for multiple animal
+  num_max_instances: 1
+  # detector network params
+
+training:
+  # select from one of several predefined image/video augmentation pipelines
+  # default- resizing only
+  # dlc- imgaug pipeline implemented in DLC 2.0 package
+  # dlc-top-down- dlc augmentations plus vertical and horizontal flips
+  imgaug: dlc-top-down
```

Comment:
fyi for some reason i found `dlc-top-down` to be worse than `dlc` for a top-down two-mouse dataset - later on i'd be interested in how these compare with your dataset

Reply:
that is weird and something I never compared! In my old code I had a L-R flip but not a top-down flip. Interested to see what matters